spec/jobs/resolution_proofing_job_spec.rb

```diff
@@ -498,6 +502,22 @@
           allow(instance).to receive(:lexisnexis_ddp_proofer).and_return(ddp_proofer)
           allow(IdentityConfig.store).to receive(:lexisnexis_threatmetrix_enabled).
             and_return(true)
+          stub_request(:post, 'https://example.com/api/session-query').
+            with(
+              body: hash_including('api_key' => 'test_api_key'),
+              headers: {
+                'Accept' => '*/*',
+                'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3',
+                'Authorization' => 'Basic Og==',
+                'Content-Type' => 'application/json',
+                'User-Agent' => 'Faraday v2.6.0',
+              },
```

Comment:
I think we can do a `hash_including` here too, user-agent and accept-encoding change often so I'd remove them

```suggestion
              headers: hash_including(
                'Authorization' => 'Basic Og==',
                'Content-Type' => 'application/json',
              ),
```